omod/src/main/java/org/openmrs/module/cohort/web/resource/CohortResource.java

```diff
@@ -224,16 +223,8 @@ public Collection<CohortAttribute> getCohortAttributes(CohortM cohort) {
 	 */
 	@PropertySetter("attributes")
 	public void setAttributes(CohortM cohort, List<CohortAttribute> attributes) {
-		if (attributes != null) {
-			User authenticatedUser = Context.getAuthenticatedUser();
```

Comment:
It would be better if we kept in the voiding logic, I think, even if we're replacing the `addAll()` call with a loop.